.github/workflows/ci.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           #
           # manual_clamp will panic when min > max
           # See https://github.com/rust-lang/rust-clippy/pull/10101
-          cargo clippy --all-features --all-targets --workspace -- -A clippy::derive_partial_eq_without_eq -A clippy::manual_clamp ${{ matrix.args }}
+          cargo clippy --all-features --all-targets --workspace -- -A clippy::derive_partial_eq_without_eq -A clippy::manual_clamp -A clippy::too_many_arguments  ${{ matrix.args }}
```

Comment:
We should probably move these settings to the `clippy.toml` so editors pick up our configuration as well

https://doc.rust-lang.org/clippy/configuration.html

Reply:
Yep was looking at that. From what I can tell only [Cargo.toml](https://doc.rust-lang.org/clippy/configuration.html#lints-section-in-cargotoml) supports the in file config. But thats is not available in the rustc version we are on. I can open an issue to migrate the lints to Cargo.toml once we upgrade toolchain.